packages/playground/src/PlaygroundImports.js

```diff
@@ -11,6 +11,7 @@ import '../../../app/hotels/src/allHotels/searchForm/locationPicker/__tests__/Su
 import '../../../app/hotels/src/allHotels/searchForm/locationPicker/__tests__/RecentSearches.test.js';
 import '../../../app/hotels/src/allHotels/__tests__/LoadMoreButton.test.js';
 import '../../../app/MMB/src/scenes/passenger/visa/__tests__/VisaRequired.test.js';
+import '../../../app/MMB/src/scenes/passenger/menu/__tests__/OtherTitledMenuGroup.test.js';
```

Comment:
This seems outdated. Thankfully... :)

Reply:
Yes, that is true 🤕 I will update the playground imports